jdi-light-mobile/src/main/java/com/epam/jdi/mobile/driver/get/MobileDriver.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.jdi.mobile.driver.get;
+
+public class MobileDriver {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'MobileDriver' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4973196)